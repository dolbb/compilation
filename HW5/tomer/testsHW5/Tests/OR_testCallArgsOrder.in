int first(){
print("1\n");
return 1;
}

int second(){
print("2\n");
return 2;
}

int third(){
print("3\n");
return 3;
}

int sum(int x,int y,int z)
{
return (x+y+z)/3;
}

void main(){
int x = sum(first(),second(),third());
print("res is ");
printi(x);
print("\n");
}