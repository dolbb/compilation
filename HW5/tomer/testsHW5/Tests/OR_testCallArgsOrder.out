Loaded: ./exceptions.s
1
2
3
res is 2
